.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
 .idea
 .DS_Store
-.vscode/launch.json
\ No newline at end of file
+.vscode/launch.json
+*.snap.new
```

Comment:
@c-git I made this more specific, which I think is fine?

Reply:
Yep that's better. In hindsight that's probably what it always should have been.